app/controllers/time.ts

```diff
@@ -204,6 +205,7 @@ export default function (controller: botkit.Controller) {
         Console.error('No Organization was found for the team: ' + bot);
         return;
       }
+      const channelName = message.channel || message.channel_obj.name;
```

Comment:
Ditto the above comment, you need to use `message.channel_obj.name` if you want the actual channel name.